solr/conf/schema.xml

```diff
@@ -308,7 +308,6 @@
       <analyzer>
         <tokenizer class="solr.StandardTokenizerFactory"/>
         <filter class="solr.StopFilterFactory" ignoreCase="true" words="stopwords.txt"/>
-        <filter class="solr.StandardFilterFactory"/>
```

Comment:
are we sure we want to change the solr schema?

Reply:
All the changes to the schema.xml files were due to the upgrade to solr8 service. Each of them were no longer supported in solr 8.